browser/branding/recordreplay/pref/firefox-branding.js

```diff
@@ -18,6 +18,9 @@ pref("app.update.url.manual", "https://replay.io");
 // supplied in the "An update is available" page of the update wizard.
 pref("app.update.url.details", "https://replay.io");
 
+// Adding a trailing # to support appending targeted help from "Learn more" links
```

Comment:
I wonder if we want to start adding something greppable like

```
// [REPLAY] Add support link
// Adding a trailing # to support appending targeted help from "Learn more" linksWrite<br class="Apple-interchange-newline">
```

Reply:
In general, I agree. This file, in particular, is a replay-specific file so I think it's redundant to call it out here.

Reply:
I did add a `// [REPLAY]` comment to a couple other places to improve (re-)discovery.